packages/portal/document-viewer/src/components/nodes/Heading.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <component :is="tag">
+  <component
+    :is="tag"
+    :id="slug"
```

Comment:
question: if we want better assurance of unique element ids across the consuming application, should we add a specific prefix to the slugs so it would be like `doc-${tag}-${slug}`? Not sure if that adds a lot of value for the tradeoff more visually complex URLs.

Reply:
We have a pattern of generating an id with the `uuid` package if needed

Reply:
After some discussion I think we should keep it human-readable, but with a docs prefix (but ability for folks to pass their own if they're into that kind of thing). Leaves us the option of putting the tag in it if we want to consume it, yeah?

Reply:
The main thing is we want a deterministic value that will persist across page loads and is derived from the content itself, to enable shareable links and consistent reloads.